core/trace/src/main/scala/org/typelevel/otel4s/trace/Tracer.scala

```diff
@@ -67,6 +67,18 @@ trait Tracer[F[_]] extends TracerMacro[F] {
         fa
     }
 
+  /** Creates a new tracing scope if a parent can be extracted from the given
+    * `carrier`. A newly created non-root span will be a child of the extracted
+    * parent.
```

Comment:
It should probably say what happens if a parent can't be extracted.

Reply:
Good point. I added an extra paragraph.